cloudbuild/nightly/scripts/python-scripts/create_unbounded_sink_table.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+from collections.abc import Sequence
+
+from absl import app
+from google.cloud import bigquery
+from absl import logging
+from google.cloud.bigquery import DatasetReference
+
+# The following operations are performed for internal, unbounded read-write tests:
+# 1. Copying source files to a temporary GCS directory which acts as a new source.
+# 2. Creating a destination table with a hardcoded schema.
+# 3. Running the Flink job in unbounded mode while dynamically adding new files to the source.
```

Comment:
Let's shift this to top of the file as file description.

Reply:
Sure. Added it to the top.